packages/server/src/course/course.module.ts

```diff
@@ -1,8 +1,9 @@
 import { Module } from '@nestjs/common';
-import { TypeOrmModule } from '@nestjs/typeorm';
 import { CourseController } from './course.controller';
+import { QueueModule } from '../queue/queue.module';
 
 @Module({
   controllers: [CourseController],
+  imports: [QueueModule],
```

Comment:
I felt that the cleanup function was better suited as service in the `QueueModule` but it could have been done in the `CouresModule` and not needed to have been imported.